### ISSUE TITLE
How to use cat-dataset

### ISSUE THREAD

**Issue by Sakakibaraki** [ESanchezLozano/SAIC-Unsupervised-landmark-detection-NeurIPS2019#4]
Hello,
I could train a model with celebA and got good results,
but I wasn't sure how to train a model using the cat-dataset in your paper.

I have already downloaded the data set.
Would you tell me how to use it?

**Comment by Sakakibaraki** [comment]
I resolved. Close the issue.

**Comment by ESanchezLozano** [comment]
Sorry for the late reply, I am glad you solved it though :)

**Comment by Sakakibaraki** [comment]
Thank you very much for your immediate reply.

I'm sorry. I thought I had solved the problem, but now I can't figure out how to do unsupervised learning of the target. ("Learning the target domain network" in chapter 3)
Is there a code somewhere that I can use to train without using the original annotations of the cat-dataset?

**Comment by ESanchezLozano** [comment]
Hi,

You don't need the original annotations whatsoever, indeed that's the point of the method being unsupervised!!

The target domain network is the one you will learn in that unsupervised way, with the cats in this case. The original domain network is the one attached to this repo that was trained for human pose estimation.

You only need raw images to train the network on the target domain (cats in this case).

I hope this helps :)

**Comment by Sakakibaraki** [comment]
Thank you for your reply.

I thought I had to prepare a file like "list_landmarks_align_celeba.txt" corresponding to the cat dataset, could I be wrong?

**Comment by ESanchezLozano** [comment]
Yes, the landmarks for the celeba are just used to crop the images without using a face detector because the images might not be cropped. For the cats dataset, if the heads are already cropped which I remember to be the case, you don't need to do that

**Comment by Sakakibaraki** [comment]
Sorry, I was trying a few things and I'm late in replying.

I think I need to define the point as well as the filename of the image for the database (in databases.py, collect() returns image, points), 
what should I specify there?
